core/schema/sdk.go

```diff
@@ -91,11 +92,22 @@ func (s *moduleSchema) builtinSDK(ctx context.Context, root *core.Query, sdkName
 		return s.loadBuiltinSDK(ctx, root, sdkName, digest.Digest(os.Getenv(distconsts.PythonSDKManifestDigestEnvName)))
 	case "typescript":
 		return s.loadBuiltinSDK(ctx, root, sdkName, digest.Digest(os.Getenv(distconsts.TypescriptSDKManifestDigestEnvName)))
+	case "community/elixir":
+		return s.sdkForModule(ctx, root, pinUpstreamModule("github.com/dagger/dagger/sdk/elixir/runtime"), dagql.Instance[*core.ModuleSource]{})
+	case "community/php":
+		return s.sdkForModule(ctx, root, pinUpstreamModule("github.com/dagger/dagger/sdk/php/runtime"), dagql.Instance[*core.ModuleSource]{})
```

Comment:
The dagger.json for the php sdk is in the sdk root, not the runtime dir so this should be: `github.com/dagger/dagger/sdk/php`

Reply:
Good point. @helderco are we okay with this being inconsistent with go/typescript/python, or should we update those/these?

Reply:
I think it's ok for now. It's consistent with community SDKs anyway. We'll eventually probably want to do the same for Py and TS too, even if just for:
- https://github.com/dagger/dagger/issues/7585